src/index.js

```diff
@@ -498,6 +499,8 @@ function createConfig(options, entry, format, writeMeta) {
 								compilerOptions: {
 									sourceMap: options.sourcemap,
 									declaration: true,
+									allowJs: useTypescript || emitDeclaration,
```

Comment:
Since the same check already happens on line 490, allowJs is always true here, personally I would replace it with `allowJs: true` 😁

Reply:
Ha! Good catch.

Reply:
```suggestion
									allowJs: true,
```